src/agent/modes.js

```diff
@@ -2,10 +2,38 @@ import * as skills from './library/skills.js';
 import * as world from './library/world.js';
 import * as mc from '../utils/mcdata.js';
 import settings from '../../settings.js'
+import translate from 'google-translate-api-x';
 
-function say(agent, message) {
+
+async function handleTranslation(message, lang) {
```

Comment:
this is a duplicate function, it should reuse a function from a shared translator file